python/src/data.py

```diff
@@ -0,0 +1,46 @@
+import getopt, sys, csv
+from bs4 import BeautifulSoup, Comment
+
+# Argument phraser
```

Comment:
It is always a good idea to add some documentation to the file:
- What is this script used for?
- Example usage
- Description of the input and output.